src/Compilers/CSharp/Portable/FlowAnalysis/DataFlowPass.cs

```diff
@@ -4021,6 +4006,19 @@ public override BoundNode VisitAsOperator(BoundAsOperator node)
             return result;
         }
 
+        public override BoundNode VisitSuppressNullableWarningExpression(BoundSuppressNullableWarningExpression node)
+        {
+            var result = base.VisitSuppressNullableWarningExpression(node);
+
+            Debug.Assert(!IsConditionalState);
+            if (_performStaticNullChecks && this.State.Reachable)
+            {
+                this.State.ResultIsNotNull = true;
```

Comment:
>this [](start = 16, length = 4)

Suggest either making it oblivious or implementing the suppression at the use site.

Reply:
Changed to oblivious (`null`).